### ISSUE TITLE
Run e2e prow tests daily

### ISSUE THREAD

**Issue by rlingutla** [GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp#103]
### TL;DR
As a result of the failed e2e test runs for [PR 102](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/pull/102), having the prow tests run continuously would help us catch some of these failures faster. 

### Design

**Proposal**
At the suggestion of @tam7t, we should be able to modify some prow configurations to set up a continuous test

**Resources**
- [Prow Settings](https://github.com/GoogleCloudPlatform/oss-test-infra/blob/c2e9a7559b851d017fe95ac60d4ff84149632cc6/prow/prowjobs/GoogleCloudPlatform/esp-v2/esp-v2.yaml#L582)
- [Secret Manager CSI Driver Prow Config](https://github.com/GoogleCloudPlatform/oss-test-infra/tree/master/prow/prowjobs/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp)

**Comment by tam7t** [comment]
Let re-open until we get a link to the testgrid

**Comment by tam7t** [comment]
it is running:

https://oss-prow.knative.dev/view/gs/oss-prow/logs/prow-e2e-tests-periodic/1396752879221477376

but failing with

```
+ export CLUSTER_NAME=management-cluster
+ CLUSTER_NAME=management-cluster
+ export PROJECT_ID=secretmanager-csi-build
+ PROJECT_ID=secretmanager-csi-build
+ export SECRET_STORE_VERSION=v0.0.18
+ SECRET_STORE_VERSION=v0.0.18
test/infra/prow/presubmit.sh: line 28: PULL_PULL_SHA: unbound variable
```

Likely need to bind some more config for the periodic run to the `main` branch
